docs/README.md

```diff
@@ -19,8 +18,21 @@ If the files in `pkg/apis` are updated we need to run `codegen` scripts
 
 ## Running Operator (Development)
 
-1. Set `KO_DOCKER_ENV` environment variable ([ko#usage](https://github.com/google/ko#usage))
+### Setup
+- Set `KO_DOCKER_ENV` environment variable ([ko#usage](https://github.com/google/ko#usage))
+
+### Run operator
+
+**Target: Kubernetes**
+```shell script
+    make apply
+```
+
+**Target Openshift**
+```shell script
+    make TARGET=openshift apply
+```
 
-1. Set `KO_DATA_PATH=${GOPATH}/src/github.com/tektoncd/operator/cmd/operator/kodata`
+## Running Tests
 
-1. run `make ko-apply`
+[test docs](../test/README.md)
```

Comment:
As you have updated README can you please add instruction for cleanup also ?
`make clean-cluster`